js/adopters_test.js

```diff
@@ -0,0 +1,83 @@
+const eclipsefdn_adopters_element = document.querySelectorAll(".eclipsefdn-adopters")[0];
+
+function testAdopters(working_group) {
+	fetch("/adopters/assets/js/adopters.json")
+	.then(response => {
+		if (response.ok) {
+			return response.json();
+		} else {
+			return Promise.reject(response);
+		}
+	})
+	.then(data => createWGAdopters(data.adopters, working_group))
+	.catch(error => console.log(error))
+}
+
+function createWGAdopters(data, working_group) {
+
+    let project_set = new Set();
+    for(let i=0; i<data.length; i++) {
+
+        // Loop the projects of this adopter
+        for(let j=0; j<data[i].projects.length; j++) {
+			let project = data[i].projects[j];
```

Comment:
Let is not a construct in IE and will likely fail. Use only vars in js (this should be caught by our linters as well)